plugin/src/Modifier.cc

```diff
@@ -927,6 +927,172 @@ Mod_As_Duration::load(Config &cfg, YAML::Node, TextView, TextView, YAML::Node ke
   return Handle(new self_type{std::move(expr)});
 }
 
+// ---
+/// url-encode modifier.
+class Mod_url_encode : public Modifier
```

Comment:
The encode/decode are quite similar, if we extend this to perform different character sets, we may pullulate the file with a lot of similar modifiers. Maybe this could be part of the  Feature(like `as_duration`, `as_bool`, etc)

Reply:
Could pass the type as an argument.
```
uri-encode<utf8>:
```